test/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/SubmitReturnsConnectorSpec.scala

```diff
@@ -56,25 +63,49 @@ class SubmitReturnsConnectorSpec extends WordSpec with Matchers with MockFactory
     )
   )
 
-  val submitReturnRequest: SubmitReturnRequest = {
-    sample[SubmitReturnRequest].copy(completeReturn = sample[CompleteReturn]
-      .copy(triageAnswers = sample[CompleteSingleDisposalTriageAnswers].copy(numberOfProperties = One))
-    )
-  }
+
 
   val connector = new SubmitReturnsConnectorImpl(mockHttp, new ServicesConfig(config, new RunMode(config, Mode.Test)))
 
   "SubmitReturnsConnectorImpl" when {
 
     implicit val hc: HeaderCarrier = HeaderCarrier()
-    val expectedHeaders            = Map("Authorization" -> s"Bearer $desBearerToken", "Environment" -> desEnvironment)
+    val expectedHeaders = Map("Authorization" -> s"Bearer $desBearerToken")//, "Environment" -> desEnvironment)
```

Comment:
aren't we getting the `Environment` header? We should be right...?

Reply:
we get it, I enabled check